docs/about_GettingStarted.md

```diff
@@ -0,0 +1,167 @@
+﻿---
+layout: default
+title: Getting Started
+type: about
+---
+
+# Getting Started
+
+## about_GettingStarted
+
+# SHORT DESCRIPTION
+
+Here is a super simple Polaris application to get you started
+
+```ps
+Import-Module Polaris
+New-PolarisGetRoute -Path "/" -Scriptblock {
+    $Response.Send('Hello World!');
+}
+
+Start-Polaris
+```
+
+The application starts a server listening on port 8080 (the default) for connections. It will respond to with "Hello World!" to requests to the root url (/) or **route** it will respond to every other request with a 404 not found.
```

Comment:
```suggestion
The application starts a server listening on port 8080 (the default port) for connections. It will:

- Respond with "Hello World!" to requests to the root url (/)
- Respond with a 404 not found to any other route (ex. `/DoesNotExist`)
```

Reply:
Feel free to modify :)